### ISSUE TITLE
Cannot connect to channels that have a password

### ISSUE THREAD

**Issue by harveyaj11** [zelloptt/zello-channel-api#20]
I can't find a way to connect to channels that are set to use a password. I can connect to other public channels fine. Is there an option that needs to be set in the var session = new ZCC.Session({… call.

**Comment by megamk** [comment]
@harveyaj11 password protected public channels are not currently supported by channel-api server

**Comment by blackwellj** [comment]
Is this a feature that can be enabled?

**Comment by MerApple** [comment]
@harveyaj11 do you think this is a feature that will be enabled in the future or is this something you never will add support for?

**Comment by vocoded** [comment]
@MerApple yes, we expect to eventually support these types of channels. Unfortunately there is no ETA.

**Comment by makegofast** [comment]
This seems like a very important feature not to include. This has been open for more than 4 years. Any intentions to actually add this feature?

**Comment by ibnux** [comment]
It's time to support password protected channels

Why if user already member password protected channel, it cannot connect using API?
